distribution/deb/src/test/java/org/elasticsearch/test/rest/ClientYamlTestSuiteIT.java

```diff
@@ -20,18 +20,21 @@
 package org.elasticsearch.test.rest;
 
 import com.carrotsearch.randomizedtesting.annotations.ParametersFactory;
-import org.elasticsearch.test.rest.parser.RestTestParseException;
+
+import org.elasticsearch.test.rest.yaml.ClientYamlTestCandidate;
+import org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase;
+import org.elasticsearch.test.rest.yaml.parser.ClientYamlTestParseException;
 
 import java.io.IOException;
 
 /** Rest integration test. Runs against a cluster started by {@code gradle integTest} */
-public class RestIT extends ESClientYamlSuiteTestCase {
-    public RestIT(RestTestCandidate testCandidate) {
+public class ClientYamlTestSuiteIT extends ESClientYamlSuiteTestCase {
```

Comment:
I *hope* that isn't a big deal. Yeah, we do have more classes named this then before because I renamed stuff like `SmokeTestsDocsIT` to this name as well.